### DISCUSSION TITLE
Ability to disable AP mode

### DISCUSSION THREAD

**Discussion by ielbury** [HASwitchPlate/HASPone#42] [Ideas]
Whenever I reboot my AP or repeaters, the plate goes into AP mode (192.168.4.1). This requires a power cycle which if it is in the wall, can be a problem. Alternatively, I have to connect to the plate AP and reboot but the problem with that is ALL the plates use the same SSID so you have no idea who you are connecting to. All of this is not wife friendly 😂

Once I have used AP mode for the initial configuration, I don't use it again.

**Comment by aderusha** [comment]
So this is actually by design - if it cannot connect to WiFi (because maybe you rebooted the WiFi), it'll try and reconnect, and then failing that will go into AP mode.  It'll sit in AP mode for 5 minutes, then reboot and try the whole thing again.  So, in your case, you might be able to just let it run for another 5 minutes and it should reconnect.

**Reply by ielbury** [reply]
I understand the design decision, I looked at the code and didn't see it reboot in 5 minutes. The bigger issue is all the plates use the same SSID ... maybe the SSID needs to be be post fixed with part of the MAC address like Tasmota?

**Reply by ielbury** [reply]
Maybe Page 0 or the page that AP mode uses could have a reboot button?

**Reply by aderusha** [reply]
In prior versions the SSID worked just like you describe, but we ran out of memory on the rest of the project for the QR code component and a decision was reached to ditch the QR and get a couple more pages out of the project.  Without being able to generate a dynamic QR, the SSID and pass need to be fixed.

However - I think the reboot idea might work.  Let me see if I can't shoehorn that into the current project somehow.

**Reply by ielbury** [reply]
Thanks Allen ... a reboot button makes it easy to train the wife 😊

**Reply by aderusha** [reply]
Got this working and it looks good.  You should see this feature in the next build of HASPone. Thanks @ielbury for the idea, I think this will work well!
